extras/log_file_server/bin/server.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:async' show Future;
+import 'dart:io';
+
+import 'package:shelf/shelf.dart';
+import 'package:shelf/shelf_io.dart';
+import 'package:shelf_router/shelf_router.dart';
+
+const rootDir = '/var/opt/oap/logs';
+
+// Configure routes.
+final _router = Router()
+  ..delete('/logs/deleteMe.log', _delete)
```

Comment:
This path is hard-coded because we don't want this functionality on a non-secure server, but we want it for tests?

Reply:
Yes, I figured that if someone uploaded data to this file and someone else deleted it I wouldn't feel bad!